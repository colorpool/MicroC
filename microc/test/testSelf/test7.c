void main(int n) {
    int i;
    i =  n>2?1:2;
    print 'd' i;
    
}