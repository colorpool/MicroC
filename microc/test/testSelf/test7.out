24 19 1 5 25 15 1 13 0 1 1 13 0 0 1 11 0 2 10 7 17 26 0 1 16 28 0 2 12 15 -1 13 0 1 1 11 22 15 -1 15 -1 21 0